SocialFilter/sections/css_extended.txt

```diff
@@ -18,7 +18,6 @@ ranking.net#?##js-top > div.flex > a[href^="https://twitter.com/intent/"]:upward
 realt.by#?#div.col > div.object-item div[class^="col"]:has(> div.ya-share2)
 voleybolmagazin.com#?#.theiaStickySidebar > div.widget_text:has(> div.textwidget > a.twitter-timeline)
 leeapk.com#?#article > div.text-center:has(> ul.list-shares)
-flashback.org#?#.dropdown-toggle:has(> span > i.fa)
```

Comment:
As I can see, this rule removes useful element, but it's not necessary to remove it.
Fixed directly.
Next time don't forget to fill out the pull request completely.